source/Generic/SteamViewer/SteamViewer.psm1

```diff
@@ -80,11 +80,24 @@ function GetGameMenuItems
         $menuItem8.FunctionName = "Start-Library"
         $menuItem8.MenuSection = $extensionName
 
-        return $menuItem1, $menuItem2, $menuItem3, $menuItem4, $menuItem5, $menuItem6, $menuItem7, $menuItem8, $menuItem9, $menuItem10, $menuItem11, $menuItem12, $menuItem13, $menuItem14
+        #check if workshop link exists before adding it
+        $workshopLink = $game.Links | Where-Object { $_.Name -eq "Workshop" }
+        if ($workshopLink)
+        {
```

Comment:
1. I'd be better if the link search was done by the url instead of name as it would prevent false positives or work in cases where the link name was changed for any reason
2. I'd prefer if the link search was done in a loop that breaks upon finding, to prevent having to loop through all the links